op-devstack/stack/context.go

```diff
@@ -9,90 +9,28 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/logfilter"
 )
 
-type kindCtxKeyType struct{}
-
-var kindCtxKey = kindCtxKeyType{}
-
-const UnknownKind Kind = ""
-
-// KindFromContext reads what the kind of service the context is focused on. This may be UnknownKind if unspecified.
-func KindFromContext(ctx context.Context) Kind {
```

Comment:
Please keep this function. The string key + generic return value, using the logging context to overload, I think is unsafe.

Reply:
Adding back.